### DISCUSSION TITLE
Wish list - store a sort order

### DISCUSSION THREAD

**Discussion by Deanobeano2** [cortexrd/Knack-Toolkit-Library#424] [General]
Hi

While testing Knack's options for scheduling, since they don't have timeline views, I noticed in a trial report I created that I can sort and group by Employee and Date which is what I want. However, I don't seem to be able to store the next level of control of the report.

![image](https://github.com/user-attachments/assets/2aaa6b06-df8d-41d2-b6c3-c70a5464e759)

Clicking the Start Time header then allows me to then sort the shifts from earliest to latest which is what I want.

![image](https://github.com/user-attachments/assets/84585dc9-8c4d-444f-81b3-96dab6903657)

rosterschedule/?view_1749_sort=field_1935-field_1928|asc

Is it possible to store the sort setting for the report? I would filter the report after this by date ranges. Or, am I missing something?

Dean

**Comment by topdog-BigDogWerks** [comment]
Were you able to work this out?
I think you would control that in Grid's Source settings.

![image](https://github.com/user-attachments/assets/b9d3a70d-2e01-4889-ac9b-d26706bfe482)

The configuration below controls a table grouped by Office location. Ignore the filters (they aren't relevant here) just look at the sort. 
This is using numeric Status Sort field and then sorting by an Alphanumeric Project Number field.

![image](https://github.com/user-attachments/assets/ed292a10-db06-45a1-b01c-4e111fe08210)
